xml/System.Net.Http/HttpRequestOptions.xml

```diff
@@ -479,8 +484,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Returns an enumerator that iterates through the collection.</summary>
+        <returns>An enumerator that can be used to iterate through the collection.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Net.Http.HttpRequestOptions> instance is cast to an <xref:System.Collections.IEnumerable> interface.

          ]]></format>
        </remarks>
```